### DISCUSSION TITLE
Query builder diagram

### DISCUSSION THREAD

**Discussion by manztihagi** [dbeaver/dbeaver#33042] [General]
I'm trying to evaluate DBEaver Enterprise Edition with version ``24.0.0.202404011634`` 

How to create query builder showing like this below:

![DBeaver_QBuilder](https://github.com/dbeaver/dbeaver/assets/37173859/e5aacdf9-6c87-48f1-a5fa-04fe31fc4f6c)

rather than :

![DBeaver_query_builder_tables0](https://github.com/dbeaver/dbeaver/assets/37173859/72fe20b9-4dc0-4048-b86c-d388fbf1a068)

Because in the first image is more clearly out the link between fields rather than second image that more harder to exactly find-out the link between table, must be read on Query joins table condition tabs.

**Comment by serjiokov** [comment]
Hello @manztihagi,thanks for the feature request, we'll discuss it and let you know of the implementation possibility.

**Reply by manztihagi** [reply]
Hello @serjiokov, I thought isn't the feature request, but already shown on your website :
link: dbeaver.com/dbeaver-enterprise/ => section Database development

<img width="960" alt="image" src="https://github.com/dbeaver/dbeaver/assets/37173859/769f6106-2dba-4a02-999c-8c9dadbfe61e">

So that's why I'm asking for

**Comment by serjiokov** [comment]
The ortho path for diagrams is a commercial feature your can try it in our pro product version.
